pgjdbc/src/test/java/org/postgresql/test/jdbc2/StatementTest.java

```diff
@@ -125,6 +127,16 @@ public void testUpdateCount() throws SQLException {
 
     count = stmt.executeUpdate("CREATE TEMP TABLE another_table (a int)");
     assertEquals(0, count);
+
+    count = stmt.executeUpdate("CREATE TEMP TABLE another_tablecount AS "
+        + "select true test from generate_series(1,25);");
```

Comment:
would suggest adding the noise word "as" between true and test

Reply:
And, the commit message should read "For a SELECT INTO or CREATE TABLE AS command", just like the release note;

https://www.postgresql.org/docs/9.0/static/release-9-0.html

E.24.3.2

The INTO is not noise and its absence will confuse people who are thinking this impacts an actual SELECT query that returns a result set.

Reply:
Done @polobo, thanks to point to the release notes.